uui/components/buttons/Button.module.scss

```diff
@@ -131,26 +131,32 @@
     &:global {
         &.uui-size-18 {
             @include button-layout(18px, 1px);
+            @include scalable-icon-size(18);
```

Comment:
Probably we can add scalable-icon-size mixin call into button-layout mixin? @dsv-sergey

Reply:
Need to investigate all places where use button-layout mixin

Reply:
it's the only place where it uses, I moved scalable-icon-size mixin into button-layout mixin, working well

Reply:
made it for LinkButton